File: src/main/java/com/official/memento/todo/domain/vo/ClaudeAiChatClientOutputPort.java
package com.official.memento.todo.domain.vo;

import com.official.memento.todo.domain.entity.ToDo;

import java.util.List;

public interface ClaudeAiChatClientOutputPort {
    List<PrioritizedToDo> prioritizeTodo(
            List<ToDo> todoList,
            List<Double> orderList,
            String personalInfo
    );
}

File: src/main/kotlin/com/official/memento/todo/domain/vo/ClaudeAiChatClientOutputPort.kt
package com.official.memento.todo.domain.vo

import com.official.memento.todo.domain.entity.ToDo

interface ClaudeAiChatClientOutputPort {
    fun prioritizeTodo(
            todoList: List<ToDo>,
            orderList: List<Double>,
            personalInfo: String
    ): List<PrioritizedToDo>
}
